src/components/react-admin/DraggableFormIterator.js

```diff
@@ -0,0 +1,282 @@
+/* eslint-disable
```

Comment:
Can you make a first commit including this copy of FormIterator with no change at all, and then another commit with the changes you made?

This would allow identifying your changes, and re-applying them in case of FormIterator original file upgrade.

Reply:
I did not start from SimpleFormIterator but from another PR https://github.com/marmelab/react-admin/pull/1836 so it has greatly diverged and it wouldn't mean anything